src/views/clusteroverview/OverviewTab/resources-inventory-card/ResourcesInventoryCard.tsx

```diff
@@ -26,7 +28,7 @@ const ResourcesInventoryCard: React.FC = () => {
             <ResourceInventoryItem
               quantity={vms}
               label={t('VirtualMachines')}
-              path={getResourceUrl(VirtualMachineModel)}
+              path={getResourceUrl(VirtualMachineModel, undefined, activeNamespace)}
```

Comment:
I recommend changing `undefined` to `null` here and in the other places you've used it in this file. This [page](https://www.geeksforgeeks.org/undefined-vs-null-in-javascript/) gives a good explanation of the reasoning. The short version is: 

> **Null**: It is the intentional absence of the value. It is one of the primitive values of JavaScript.
> **Undefined**: It means the value does not exist in the compiler. It is the global object.

Reply:
Well, in some other resources I've found, undefined is recommended to omit a specific optional parameter if there are more of them. But I will reconsider this. Thanks!

Reply:
[This](https://stackoverflow.com/questions/33232066/do-you-have-to-give-all-parameters-when-firing-a-function-in-javascript) supports that. I had never heard that, so thanks for teaching me something new. =)

Reply:
To avoid null or undefined we can change getResourceUrl , to receive and obj instead of separate args.

Reply:
Yes. I was thinking about creating an interface for `getResourceUrl` to accept an object instead, but it does not seem to be very useful for other cases than this one in here..

Reply:
> Yes. I was thinking about creating an interface for `getResourceUrl` to accept an object instead, but it does not seem to be very useful for other cases than this one in here..

other cases will need to be edited into an obj, but this makes the function much nicer and also if in the future it will need to receive another arg make it much faster and easier.

Reply:
if we can make our code base better we should try :)

Reply:
See the followup PR: https://github.com/kubevirt-ui/kubevirt-plugin/pull/962